doc/reference/reference_lua/box_null.rst

```diff
@@ -0,0 +1,83 @@
+.. _box-once:
+
+-------------------------------------------------------------------------------
+                             Constant `box.NULL`
+-------------------------------------------------------------------------------
+
+There are some major problems with using a Lua **nil** value in tuples.
+For example: you can't correctly assess length of sparse array (an array with
+nullable values set to null) in case of using Lua **nil** value.
+
+**Example:**
+
+.. code-block:: tarantoolsession
+
+    tarantool> t = {0, nil, 1, 2, nil}
+    ---
+    ...
+
+    tarantool> t
+    ---
+    - - 0
+      - null
+      - 1
+      - 2
+    ...
+
+    tarantool> #t
+    ---
+    - 4
+    ...
+
+To overcome this and some other issues use ``box.NULL`` constant instead.
+
+Using box.NULL
+--------------
+
+``box.NULL`` is a generic pointer to a C-like NULL. So it is
```

Comment:
Bad definition. As a user I don't get it. Also, what is "C-like NULL", and what is pointer it?

Reply:
Reworked the definition.